src/plonk/circuit.rs

```diff
@@ -1147,10 +1148,15 @@ impl<F: Field> ConstraintSystem<F> {
         index
     }
 
-    fn query_any_index(&mut self, column: Column<Any>, at: Rotation) -> usize {
+    fn query_any_index<C: Into<Column<Any>>>(&mut self, column: C, at: Rotation) -> usize {
+        let column = column.into();
```

Comment:
Hmm, this didn't actually need to be altered, because `query_any_index` is a private method that is only called by `enable_equality`, which already performs the `.into()`. Undo this change (if it is necessary later, we can do it then).
```suggestion
    fn query_any_index(&mut self, column: Column<Any>, at: Rotation) -> usize {
```